Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import Button from './components/ButtonComponents/NumberButton';
 
 const App = () => {
   return (
-    <div>
+    <div className="container">
```

Comment:
As long as you have one outermost container (div), you can remove any extraneous divs that aren't being used.